### ISSUE TITLE
Generate hdbsynonyms along with CDS for massConvert operation

### ISSUE THREAD

**Issue by ThePlenkov** [SAP-samples/hana-developer-cli-tool-example#48]
Hi!

One more suggestion which I would like to come with:

Right now when we generate schema CDS with `hana-cli massConvert` operation we have CDS with renamed tables/views. 

However after HDI deployment to consume CAP CDS views we will also need .hdbsynonyms file containing all those views. If it's like couple views - it's OK to recreate, however when we have large schemas, especially in cross-schema projects - the amount of manual work is quite high.

That's why I just thought - what if we generate also synonym file along with cds file:
```
hana-cli mc SFLIGHT --filename sflight.cds --synonyms sflight.hdbsynonyms
```

It's quite simple to save this file as we have all the data from SCHEMA

**Comment by ThePlenkov** [comment]
I'll provide in MR

**Comment by jung-thomas** [comment]
That's an excellent idea. I could imagine people even wanting to use to support calculation view scenarios and just save time when they need a lot of synonym access to a foreign schema.

**Comment by jung-thomas** [comment]
Hope you don't mind but I made an addition here.  I noticed that synonyms were only be written out when there was some modification to the table name:
	// if modified real table names will be stored in synonyms

I added a path that will give you synonyms (if you choose that option) regardless.  For instance I might be pointing to a traditional DB Schema and want to mass generate the synonyms even if there is no name manipulation.
